public/scripts/scrapers.js

```diff
@@ -462,7 +467,9 @@ class YouTubeScraper {
         this.description = 'Download a transcript from a YouTube video.';
         this.iconClass = 'fa-brands fa-youtube';
         this.iconAvailable = true;
+    }
 
+    async init() {
```

Comment:
Is there any disadvantage to make a consistent async pattern for an interface-like function that can be extended/used by implementations of the scrapers?

If not, I feel like making all of it async is a lot more cleaner, even if not needed at the moment.
We both know how difficult it can get to refactor something later to async, when its entangled too deep.